README.md

```diff
@@ -34,7 +34,9 @@ You will also need a copy of the [App Engine SDK](https://developers.google.com/
 
 If you want to make changes to the source, you are going to need to run an automated test suite, and for that you will also need [phantomjs](http://phantomjs.org/). Have a look at the testing section for more information.
 
-Note: Java 8 will work locally, and it is also supported by AppEngine, but you may need to [specify your runtime](https://cloud.google.com/appengine/docs/standard/java/runtime-java8) if you want to use appspot.
+Note 1: If you are working on a 64-bit linux systems, you need to install 32-bit versions of: glibc(to get a 32-bit version of ld-linux.so), zlib and libstdc++.
```

Comment:
1. Either "a 64-bit Linux system" or "64-bit Linux systems"
2. Add a code block that includes the appropriate library names, e.g. ```apt-get install zlib1-dev:i386 libstdc++:i386```

Reply:
I am using Fedora, so is it okay to write the command I used for Fedora?
I can add following line after line 37:
"For e.g., if you are on Fedora, use following command: sudo dnf install glibc.i686 zlib.i686 libstdc++.i686."
I am unaware of latest versions supported by Ubuntu.

Reply:
[This file](https://github.com/mit-cml/appinventor-sources/blob/7b318cd1963bcd41690cc4a87f5ae7aa0b733c8c/bootstrap.sh#L8) has an example for the i386 for Debian/Ubuntu. You can also include the Fedora version.